src/cli.rs

```diff
@@ -169,6 +169,25 @@ pub struct CompileParameters {
     )]
     pub linker: Option<String>,
 
+    #[clap(
+        name = "debug",
+        long,
+        short = 'g',
+        help = "Generate full debug information",
```

Comment:
clang sais: "Generate source-level debug information" 
i would stick to clangs phrase